strongbox-cron/strongbox-cron-api/src/main/java/org/carlspring/strongbox/cron/jobs/CronJobDuplicationStrategy.java

```diff
@@ -0,0 +1,15 @@
+package org.carlspring.strongbox.cron.jobs;
+
+import org.carlspring.strongbox.cron.domain.CronTaskConfigurationDto;
+
+import java.util.Collection;
+
+/**
+ * @author Przemyslaw Fusik
+ */
+public interface CronJobDuplicationStrategy
+{
+
+    boolean duplicates(CronTaskConfigurationDto candidate,
```

Comment:
it will be good to have some Java Docs here.

Reply:
👍